packages/core-js/modules/es.number.constructor.js

```diff
@@ -62,7 +64,8 @@ var toNumber = function (argument) {
 
 // `Number` constructor
 // https://tc39.es/ecma262/#sec-number-constructor
-if (isForced(NUMBER, !NativeNumber(' 0o1') || !NativeNumber('0b1') || NativeNumber('+0x1'))) {
+var shouldForce = !NativeNumber(' 0o1') || !NativeNumber('0b1') || NativeNumber('+0x1');
+if (IS_PURE || isForced(NUMBER, shouldForce)) {
```

Comment:
In the case of the pure version, in environments that support the required `Number` constructor behavior, complex string parsing should not be used.

Reply:
You are right. This was my first thought. I just got confused by the fact that the node unit tests then run on the native builtin, rather than on the polyfill.

Reply:
I believe that I have addressed your concern in my last commit. Is there anything other I can do about this PR?
Or is my approach just plain wrong that you don't want to waste your time trying to correct it? :-)

Reply:
Hey, give me some time - I also have other tasks -) Now I see that this is not a completely correct approach, I'll explain the details a little later.

Reply:
You have as much time as you want. Didn't mean to be bratty.

Reply:
With the second your commit, in case of correct native `Number` behavior, in the pure version would be exported native `Number` - that means native `Number` pollution on polyfilling static `Number` properties. Also, there were some issues with reexport of statics.

For cases like this, the `export` helper has a `wrap` option.

However, in the pure version, it does not ensure proper `.name`, `.length`, and (only in some specific cases like this) subclassing - it's accepted limitations of the pure version.

Reply:
Of course, such a silly mistake, I'm sorry about that.
I totally forgot about the static properties :(